frontend/modules/shared/ui/src/lib/layout/language-switcher/language-switcher.html

```diff
@@ -0,0 +1,20 @@
+<tui-select
+  [formControl]="language"
+  [tuiTextfieldLabelOutside]="true"
+  tuiTextfieldSize="m"
+  [valueContent]="flagContent"
+>
```

Comment:
You can add `appearance="outline"` to remove box shadow, there's already shadow on whole top-bar so I don't think we need more

Reply:
good idea 💯